### PR TITLE
Use lerna and cssnext

### PR DESCRIPTION
**Pull request by xcatliu**

### CODE REVIEW

**Comment by cht8687** [comment]
Travis failed:
$ rake
rake aborted!
No Rakefile found (looking for: rakefile, Rakefile, rakefile.rb, Rakefile.rb)

@xcatliu

**Comment by xcatliu** [comment]
@cht8687 Please ignore this. The new version doesn't require ruby or sass anymore.

File: README.md [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
+
+## Features
+
+### Lightweight
+
+The default bundle of Mobi.css provides the basic styles to build clean and elegant webpages. It's **only 3kb** after gzipped, and it's very easy to get started.
+
+### Scalable
+
+Plugin system makes css scalable. You can import plugins to enhance Mobi.css, or use the plugin as a standalone css library. **It's even possible to use a plugin without Mobi.css**.
```
Comment (cht8687) [inline README.md]: really minor => css need to be capital letter in readme: CSS.
(I checked Bootstrap, Foundation... they all use capital letter CSS 😄 )

**Comment by xcatliu** [comment]
@cht8687 Fixed travis issue.

File: README.md (hunk 1 above)
Comment (xcatliu) [reply README.md]: Fixed!

File: package.json [hunk 2]
```diff
@@ -1,67 +1,34 @@
...
+		"gulp-insert": "^0.5.0",
+		"gulp-postcss": "^7.0.0",
+		"gulp-rename": "^1.2.2",
+		"gulp-sourcemaps": "^2.6.1",
+		"lerna": "^2.1.2",
+		"postcss-cssnext": "^3.0.2",
+		"postcss-import": "^10.0.0",
+		"rimraf": "^2.6.1",
+		"stylelint": "^8.0.0",
+		"stylelint-config-standard": "^17.0.0"
```
Comment (cht8687) [inline package.json:36]: missing author, repo, license info which a npm install will generate warnings.

Comment (cht8687) [reply package.json]: shouldn't we add lerna a dev dependency?

Comment (xcatliu) [reply package.json:36]: Fixed, thanks!

**Pull request merged**